astro.config.mjs

```
@@ -4,10 +4,12 @@
 import icon from 'astro-icon'
 import react from '@astrojs/react'
 import sitemap from '@astrojs/sitemap'
+import vercel from '@astrojs/vercel';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'vercel'. Consider to delete it or rename it.

'vercel' is defined here:


(lint/suspicious/noRedeclare)